TestFindCache unit test failure

seen in #1240 : 
https://circleci.com/gh/grafana/metrictank/10870?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

```
=== RUN   TestFindCache

  when findCache is empty 
    0 results should be returned ✔✔
    when adding entries to the cache ✔✔✔
      when getting cached pattern ✔✔
        After invalidating path that matches pattern ✔✔✔✔✔✔
        After invalidating path that doesnt matche cached pattern ✔✔✔✔
      when findCache invalidation falls behind ✔
        when adding to cache in backoff ✔✔✔✔✔✔✘


Failures:

  * /go/src/github.com/grafana/metrictank/idx/memory/find_cache_test.go 
  Line 71:
  Expected: '0'
  Actual:   '1'
  (Should be equal)


34 total assertions

--- FAIL: TestFindCache (0.01s)
```

--- a/idx/memory/find_cache_test.go
+++ b/idx/memory/find_cache_test.go
@@ -1,7 +1,6 @@
 package memory
 
 import (
-	"sync"
 	"testing"
 	"time"
 
@@ -56,17 +55,8 @@ func TestFindCache(t *testing.T) {
 				c.Add(1, "foo.{a,b,c}*.*", results)
 				c.Add(1, "foo.{a,b,e}*.*", results)
 				c.Add(1, "foo.{a,b,f}*.*", results)
-				c.Lock()
-				var wg sync.WaitGroup
-				wg.Add(10)
-				for i := 0; i < 10; i++ {
-					go func() {
-						c.InvalidateFor(1, "foo.baz.foo.a.b.c.d.e.f.g.h")
-						wg.Done()
-					}()
-				}
-				c.Unlock()
-				wg.Wait()
+				c.newSeries[1] <- struct{}{}
+				c.InvalidateFor(1, "foo.baz.foo.a.b.c.d.e.f.g.h")
 
 				So(len(c.cache), ShouldEqual, 0)
 				Convey("when adding to cache in backoff", func() {

